### PR TITLE
fix: Resolve the PostgreSQL error during server startup by adding missing sequences - EXO-62184

### PR DESCRIPTION
**Pull request by sofyenne**
This change is going to resolve the PostgresSQL error during the server startup by adding the messing sequences.

### CODE REVIEW

File: component/core/src/main/resources/db/changelog/social-rdbms.db.changelog-1.0.0.xml [hunk 1]
```diff
@@ -888,5 +888,9 @@
     <changeSet author="social" id="1.0.0-88" dbms="hsqldb">
         <createSequence sequenceName="SEQ_SOC_LABELS_ID" startValue="1"/>
     </changeSet>
+    <changeSet author="social" id="1.0.0-89" dbms="hsqldb,oracle,postgresql">
```
Comment (ahamdi) [inline component/core/src/main/resources/db/changelog/social-rdbms.db.changelog-1.0.0.xml]: No need to add hsqldb as sequences are already added for that DB, and we do no more support Oracle

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: component/core/src/main/resources/db/changelog/social-rdbms.db.changelog-1.0.0.xml (hunk 1 above)
Comment (sofyenne) [reply component/core/src/main/resources/db/changelog/social-rdbms.db.changelog-1.0.0.xml]: fixed

**Comment by ahamdi** [comment]
@sofyenne could you fix the conflict ?

**Comment by sofyenne** [comment]
> 

@ahamdi  resolved !

**Review by ahamdi — APPROVED** [review]

**Pull request merged**